### ISSUE TITLE
WSL 2 suddenly stopped working after Windows Update (claims missing feature)

### ISSUE THREAD

**Issue by mattrossman** [microsoft/WSL#7169]
### Windows Build Number

Microsoft Windows [Version 10.0.19041.1083]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

I can't run that command: "Invalid command line option: --status"

### Distro Version

Ubuntu 20.04

### Repro Steps

Open Windows Terminal

_Note: I have previously followed the [install instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and used WSL 2 regularly for months prior to today_

### Expected Behavior

I should see the terminal for my Ubuntu 20.04 machine running via WSL 2

### Actual Behavior

This error:
```
Please enable the Virtual Machine Platform Windows feature and ensure virtualization is enabled in the BIOS.
For information please visit https://aka.ms/wsl2-install

[process exited with code 4294967295]
```

I verified that Intel Virtualization Technology is enabled in my BIOS and re-ran the steps to enable WSL, but I still cannot access my Ubuntu terminal.

**Comment by craigloewen-msft** [comment]
If you go to "Turn Windows Features on Or Off" is the "Virtual Machine Platform" optional component enabled?

![image](https://user-images.githubusercontent.com/42221804/125492947-d2102f78-44fd-4c67-8815-07b48cc2adb8.png)

And could you please add logs for this issue? /logs

**Comment by mattrossman** [comment]
Yep, as is "Windows Subsystem for Linux". Full screenshot of my "Turn Windows Features on Or Off" panel
[wsl.etl.zip](https://github.com/microsoft/WSL/files/6812295/wsl.etl.zip)
:

![image](https://user-images.githubusercontent.com/22670878/125532421-eac35429-3de5-4e06-be86-f24b0a15a491.png)

I've attached the log recorded while I run the `wsl` command.

**Comment by pkejval** [comment]
What if you try to enable Hyper-V feature?

**Comment by mattrossman** [comment]
@pkejval Same result with Hyper-V enabled. I've tried turning on and off all of the features related to virtualization (Windows Subsystem for Linux, Virtual Machine Platform, Hyper-V, Hypervisor) and rebooting several times, none of them get rid of the error.

**Comment by mattrossman** [comment]
Also here's a screenshot of my BIOS showing virtualization is enabled. I have tried it with and without VT-D enabled with no success.

![MSI_SnapShot](https://user-images.githubusercontent.com/22670878/125881303-cac7c334-4c9b-467a-87fc-189133dfbe1b.jpg)

**Comment by mattrossman** [comment]
I had to reinstall Windows in order to get this working, so it definitely wasn't a hardware or BIOS issue. I previously had used Docker and Virtualbox on this machine, and I've read that these can cause problems with WSL 2, though they were not installed at the time that I began having this issue. Not sure if I can really consider this issue solved because reinstalling Windows is kind of a nuclear option for most users.
